flag/service/aws/aws.go

```diff
@@ -8,4 +8,5 @@ type AWS struct {
 	AccessKey     accesskey.AccessKey
 	HostAccessKey accesskey.AccessKey
 	PubKeyFile    string
+	Region        string
```

Comment:
Region flag is needed by the alerter. For the operator it comes from the custom object.